light_control.h

```diff
@@ -0,0 +1,67 @@
+#ifndef light_control_h
+#define light_control_h
+
+#include <Adafruit_NeoPixel.h>
+#include <time.h>
+
+#define SENSOR_PIN  6
+#define PIN 8   // light control pin
+#define PIN_NUM 2 //count of leds
+
+#define WAIT_FOR_SOMEONE 0
```

Comment:
One more point: state defines better be an `enum` or `enum class` in case of compiler support.

Reply:
Yes, it will be more clear)